server/src/controllers/folders.controller.ts

```diff
@@ -41,7 +41,9 @@ export class FoldersController {
     if (!ability.can('updateFolder', Folder)) {
       throw new ForbiddenException('You do not have permissions to perform this action');
     }
-    const folder = await this.foldersService.update(id, folderName);
+    const createFolderDto = new CreateFolderDto();
```

Comment:
Should do the changes here
https://github.com/ToolJet/ToolJet/pull/4849/files#diff-40c2ee9b56c32bbadbd17595c66c93b6689b59e2431284a343b14f3ea69abe81L38
body should be accepted as a `createFolderDto` object

Reply:
I think I get you for both these comments! Thank you! Still trying to get my head round DTOs